### ISSUE TITLE
false positive with `ngrx/no-reducer-in-key-names`

### ISSUE THREAD

**Issue by tarlepp** [timdeschryver/eslint-plugin-ngrx#91]
If I use something like;

```
StoreModule.forRoot(
  reducers,
  {
    metaReducers,
  }
```

In my main module I get that error - but that key is actually used on store itself - so I think that in this special case this should be skipped.

![image](https://user-images.githubusercontent.com/595561/110363255-ff0e6200-804a-11eb-837a-15165fa72ef9.png)

**Comment by timdeschryver** [comment]
This is a 🐛.
Thanks for the report @tarlepp !

**Comment by tarlepp** [comment]
@timdeschryver great! thanks alot for quick fix!
